app/scripts/modules/ecs/src/serverGroup/configure/wizard/serviceDiscovery/ServiceDiscovery.tsx

```diff
@@ -170,11 +198,21 @@ export class ServiceDiscovery extends React.Component<IServiceDiscoveryProps, IS
             <table className="table table-condensed packed tags">
               <thead>
                 <tr>
-                  <th style={{ width: '75%' }}>
+                  {useTaskDefinitionArtifact && <th style={{ width: '30%' }}>
+                    Container name
+                    <HelpField id="ecs.loadBalancedContainer" />
```

Comment:
Here we should use new help field specific to the container name field on the service registry obj. Let's use the text from the [ECS docs](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ServiceRegistry.html): "The container name value, already specified in the task definition, to be used for your service discovery service."

FYI help fields are defined in [ecs/src/ecs.help.ts](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/ecs/src/ecs.help.ts)

Reply:
Good catch Allison 👍🏼